README.md

```diff
@@ -208,6 +217,12 @@ Other possible configurations are:
 - Two internal load balancer
 - Disable default service completely `controller.service.enabled: false`
 
+### AWS and PROXY Protocol
+
+Since version 2.17, this chart has the `use-proxy-procotol` enabled by default when installed in AWS. 
```

Comment:
This also was the fact before, it just came from a different source. Back then the `cluster-operator` set it via either `ingress-controller-values` or `cluster-values` (not sure about this).